### PR TITLE
[WIP] Implements export view to excel button

### PR DESCRIPTION
**Pull request by PabloSichert**
#1197

### CODE REVIEW

File: src/components/header/SubHeader.js [hunk 1]
```diff
@@ -218,6 +224,9 @@ class Subheader extends Component {
             } while (currentNode && currentNode.children && (currentNode.type !== 'window'));
         }
 
+        // TODO: refine gridView conditional
+        const gridView = parseInt(windowType) === 143;
```
Comment (teosarca) [inline src/components/header/SubHeader.js]: atm i would suggest to not enforce it at all. Just let it be enabled for any view.

File: src/components/header/SubHeader.js [hunk 2]
```diff
@@ -256,6 +264,19 @@ class Subheader extends Component {
                         {keymap.GLOBAL_CONTEXT.NEW_DOCUMENT}
                     </span>
                 </div>}
+                {gridView && query && (
+                    <a
+                        className="subheader-item js-subheader-item"
+                        href={`/rest/api/documentView/${windowType}/${query.viewId}/export/excel?selectedIds=${selected.join(',')}`}
```
Comment (teosarca) [inline src/components/header/SubHeader.js]: the URL is not OK. See
https://github.com/metasfresh/metasfresh-webui-frontend/blob/b55753626808dcac518c94d0972ab88cd0f9a940/config.js.dist#L2

Comment (PabloSichert) [reply src/components/header/SubHeader.js]: I took the URL schema from your [comment](https://github.com/metasfresh/metasfresh-webui-frontend/issues/1197#issue-257328355) - or do you refer to explicitly prepending `config.API_URL`? Maybe I'm overlooking something here.

File: src/components/header/SubHeader.js (hunk 1 above)
Comment (PabloSichert) [reply src/components/header/SubHeader.js]: Another option would be to only enable the button if `selected.length > 0`. Or do you want the button the be there when nothing is selected (better discoverabilty of that feature)?

File: src/components/header/SubHeader.js (hunk 2 above)
Comment (PabloSichert) [reply src/components/header/SubHeader.js]: Oh, got it - I need to prepend `config.API_URL` and strip `/rest/api` from the hardcoded string.

**Comment by PabloSichert** [comment]
Thanks for looking over the changes. Updated accordingly to your comments.

**Comment by teosarca** [comment]
thanks!

**Pull request merged**